src/components/ProposeForm/FormFields/DateTime.tsx

```diff
@@ -60,10 +60,10 @@ const DateTime = ({
           type="number"
           name="duration"
           onChange={(e) => handleChange("duration", count, e.target.value)}
-          placeholder={"duration (in hours)"}
+          placeholder={"Duration (in hours)"}
           required
           className="rounded-lg font-primary focus:border-primary focus:ring-primary"
-          defaultValue={session?.duration}
+          // defaultValue={session?.duration}
```

Comment:
oo why is this commented?

Reply:
By default there was 1 in the text field, which was hiding hint text, without hint text user won't know what that 1 does. So the user would have to erase it and see the hint and then write the event duration.